frontend/tailwind.config.js

```diff
@@ -67,6 +67,14 @@ module.exports = {
       borderRadius: {
         '4xl': '2rem',
       },
+      gridTemplateColumns: {
+        'auto-1fr': 'auto 1fr',
+        '1fr-auto': '1fr auto',
+      },
+      backgroundImage: {
+        'radial-dark-green':
+          'radial-gradient(63.59% 95.05% at 42.99% 35.35%, #316146 0%, #073525 100%)',
```

Comment:
The “at syntax” is not supported by Safari: https://developer.mozilla.org/en-US/docs/Web/CSS/gradient/radial-gradient()#browser_compatibility

Reply:
While testing it seems to work fine in Safari 14 (macOS) and on the iOS Simulator (12.4, iPhone 5s). 

I've added a default dark-green background to the respective container using this radial background, in case the gradient fails. I could remove the `at` part anyway though, visually the change wouldn't be that significant.

Reply:
I'm surprise this works in Safari. Does Tailwind modify the value? Maybe it removes the “at”?

I'm not 100% sure about it but I believe you can write the exact same gradient without the “at syntax”.

Reply:
I haven't found a quick way of doing so, I'm not sure it's worth spending so much time on it (for now, at least).

Reply:
Let's move on then. You tested it and it worked.